frontend/public/components/dashboard/project-dashboard/details-card.tsx

```diff
@@ -11,12 +11,14 @@ import { getName, getRequester, GreenCheckCircleIcon } from '@console/shared';
 import { LabelList, resourcePathFromModel } from '../../utils';
 import { ProjectModel } from '../../../models';
 import { ProjectDashboardContext } from './project-dashboard-context';
+import './details-card.scss';
 
 export const DetailsCard: React.FC = () => {
   const { obj } = React.useContext(ProjectDashboardContext);
   const keys = _.keys(obj.metadata.labels).sort();
   const labelsSubset = _.take(keys, 3);
   const firstThreelabels = _.pick(obj.metadata.labels, labelsSubset);
+  const description = obj?.metadata?.annotations?.['openshift.io/description'];
```

Comment:
This would blow up above if `obj` is null.

```suggestion
  const description = obj.metadata.annotations?.['openshift.io/description'];
```